modules/integration_aws-cwagent/conf/readme.yaml

```diff
@@ -0,0 +1,3 @@
+documentations:
+
```

Comment:
there is really no documentation?

Reply:
Thank you for your comment.
I hadn't seen that readme, the main one being generated.
I've compared it with the other detectors and I think it's compliant now. Is it good for you?